en/debug/eclipse_jlink.md

```diff
@@ -0,0 +1,83 @@
+# MCU Eclipse/J-Link Debugging for PX4
+
+This topic explains how to setup and use [MCU Eclipse](https://gnu-mcu-eclipse.github.io/) with a *Segger Jlink adapter* to debug PX4 based-devices.
+
+
+## Required Hardware
+
+- [J-Link EDU Mini](https://www.segger.com/products/debug-probes/j-link/models/j-link-edu-mini/)
+- Pixhawk Debug Adapter Board
+- USB micro cable
+- Suitable cables to connect your target.
+
+## Installation
+
+### PX4
+
+Setup PX4 by following the normal guidelines:
+- [Setup the PX4 Developer Environment/Toolchain](../setup/dev_env.md) for your platofrm (e.g. for Linux see: [Development Environment on Ubuntu LTS / Debian Linux](../setup/dev_env_linux_ubuntu.md)).
+- [Download PX4](../setup/building_px4.md) and optionally build it on the command line.
+
+### Eclipse
+
+To install *Eclipse*: 
+1. Download [Eclipse CDT for C/C++ Developers](https://github.com/gnu-mcu-eclipse/org.eclipse.epp.packages/releases/) (MCU GitHub)
+1. Extract the Eclipse folder and copy it anywhere (there is no need to run any install scripts).
+1. Run Eclipse and choose a place for your initial workbench.
+
+### Segger Jlink Tools
+
+To install the *Segger Jlink* tools:
+1. Download and run the [J-Link Software and Documentation Pack](https://www.segger.com/downloads/jlink/#J-LinkSoftwareAndDocumentationPack) for your OS (Windows and Linux packages available).
+   - On Linux the tools are installed in **usr/bin**.
+
+For more information, see: https://gnu-mcu-eclipse.github.io/debug/jlink/install/
+
+## First Use
+
+1. Run *Eclipse*.
```

Comment:
Do we have to connect up the vehicle before this? Is there a particular port we need to use for the JLink connection?

Reply:
Yes defentley, yo need to power your board and connect jlink to the host and the target(px4 running device)